### PR TITLE
[SuperMinor] faster microGPT example by default

### PR DESCRIPTION
**Pull request by blefaudeux**
## What does this PR do?
Tweaking the defaults in the examples to get a nicer experience:
- commented out anomaly detection and gradient clipping, still there for experiments but reduce speed significantly (discussed with @SeanNaren and @tchaton, proposed https://github.com/PyTorchLightning/pytorch-lightning/issues/11053 but that would be some work
- gelu <> squared_relu faster for a similar accuracy, also well supported by our layers

## Before submitting

- [ ] Did you have fun?
  - Make sure you had fun coding 🙃
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
  - [ ] N/A
- [ ] Did you make sure to update the docs?
  - [ ] N/A
- [ ] Did you write any new necessary tests?
  - [ ] N/A
- [ ] Did you update the [changelog](https://github.com/facebookresearch/fairscale/blob/master/CHANGELOG.md)? (if needed)
  - [ ] N/A

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

**Comment by blefaudeux** [comment]
probably not worth it, dropping the PR
